packages/sanity/src/core/releases/tool/detail/__tests__/ReleaseSummary.test.tsx

```diff
@@ -0,0 +1,220 @@
+  useDocumentPreviewValues: ({document}: {document: SanityDocument}) => ({
+    previewValues: {title: document?.title},
+  }),
+}))
+
+jest.mock('../../../../store', () => ({
+  ...(jest.requireActual('../../../../store') || {}),
+  useUser: jest.fn().mockReturnValue([{}]),
+}))
+
+jest.mock('../../../../user-color', () => ({
+  useUserColor: jest.fn().mockReturnValue('red'),
+}))
+
+const timeNow = new Date()
+
+const releaseDocuments = [
+  {
+    _id: '123',
+    _type: 'document',
+    // 3 days ago
+    _createdAt: new Date(timeNow.getTime() - 24 * 60 * 60 * 1000 * 3).toISOString(),
+    // 2 days ago
+    _updatedAt: new Date(timeNow.getTime() - 24 * 60 * 60 * 1000 * 2).toISOString(),
+    _version: {},
+    _rev: 'abc',
+    title: 'First document',
+  },
+  {
+    _id: '456',
+    _type: 'document',
+    // 24 hrs ago
+    _createdAt: new Date(timeNow.getTime() - 24 * 60 * 60 * 1000).toISOString(),
+    // 12 hrs ago
+    _updatedAt: new Date(timeNow.getTime() - 12 * 60 * 60 * 1000).toISOString(),
+    _version: {},
+    _rev: 'abc',
+    title: 'Second document',
+  },
+]
+
+const renderTest = async (props: Partial<ReleaseSummaryProps>) => {
+  const wrapper = await createWrapper({
+    config: {
+      projectId: 'test',
+      dataset: 'test',
+      name: 'default',
+      schema: {
+        types: [
+          defineType({
+            type: 'document' as const,
+            name: 'test',
+            title: 'Test',
+            fields: [
+              {
+                type: 'string',
+                name: 'title',
+                title: 'Title',
+              },
+            ],
+          }),
+        ],
+      },
+    },
+  })
+
+  return render(
+    <RouterProvider
+      state={{}}
+      onNavigate={jest.fn()}
+      router={route.create('/test', [route.intents('/intent')])}
+    >
+      <ReleaseSummary
+        documents={releaseDocuments}
+        documentsHistory={{
+          '123': {
+            createdBy: 'created-author-id-1',
+            lastEditedBy: 'edited-author-id-1',
+          } as DocumentHistory,
+          '456': {
+            createdBy: 'created-author-id-2',
+            editors: ['edited-author-id-1', 'edited-author-id-2'],
+          } as DocumentHistory,
+        }}
+        collaborators={['author-id', 'collaborator-id']}
+        release={
+          {
+            title: 'Release title',
+            description: 'Release description',
+            _createdAt: timeNow.toISOString(),
+            authorId: 'author-id',
+          } as BundleDocument
+        }
+        validation={{}}
+        {...props}
+      />
+    </RouterProvider>,
+    {
+      wrapper,
+    },
+  )
+}
+
+describe('ReleaseSummary', () => {
+  beforeEach(async () => {
+    jest.clearAllMocks()
+
+    await renderTest({})
+  })
+
+  it('lists the release title and description', () => {
+    screen.getByText('Release title')
+    screen.getByText('Release description')
```

Comment:
just to double check, I think these tests need an expect statement right?
A getBy will work but it's not something we should likely rely on 🤔

Reply:
These tests are ok without an expect I believe. `getBy...` will throw if the element doesn't exist and fail the test. This is a pattern I remember being lauded many years ago when `testing-library/react` was first called `react-testing-library`. `.toBeInDocument` of course also works

Reply:
huh 🤔 
Maybe it's a playwright thing (and not a jest / unit thing), but I know from their guidelines that they heavily suggest using expecting